examples/policies/keyless-attestation-sbom-spdxjson.yaml

```diff
@@ -32,6 +32,9 @@ spec:
   - name: keyless
     keyless:
       url: "https://fulcio.sigstore.dev"
+      identities:
```

Comment:
Hm, so this is a fine thing to included in the examples, thank you! However, the fact that tests are not passing seems to indicate that there's a bug somewhere since missing identities should really be `warning` and not an error, for example here:
https://github.com/sigstore/policy-controller/pull/188

Let me go look.

Reply:
Yeah, there was a bug :)

Reply:
ok! cool